flutter_local_notifications/lib/src/flutter_local_notifications_plugin.dart

```diff
@@ -247,4 +248,13 @@ class FlutterLocalNotificationsPlugin {
     return FlutterLocalNotificationsPlatform.instance
         ?.pendingNotificationRequests();
   }
+
```

Comment:
Realised API docs will be needed here but I can add to this PR once other changes are made and before I merge it in